test/regressions/fixtures/CssBaseline/JoyCssBaseline.js

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { CssVarsProvider } from '@mui/joy/styles';
+import Box from '@mui/joy/Box';
+import CssBaseline from '@mui/joy/CssBaseline';
+
+export default function JoyCssBaseline() {
+  return (
+    <CssVarsProvider>
+      <CssBaseline enableColorScheme />
```

Comment:
On the GitHub issue about v6 we have a line about either we should make the color scheme opt-out or not.
In MUI Toolpad, we had to turn it on in https://github.com/mui/mui-toolpad/pull/1100. Would it make sense to start opt-out for Joy UI? And if it's a successful move, do the same for Material UI.

Reply:
Yep, that's a good idea.

Reply:
Updated.